src/pages/Signin/Signin.css

```diff
@@ -0,0 +1,44 @@
+.auth-section {
+  max-width: 30rem;
+  padding: 1rem;
+  margin: 2rem auto;
+}
+
+.auth-heading {
+  font-size: 2.6rem;
+  text-align: center;
+  margin-bottom: 2rem;
+}
+
+.input-group {
+  display: flex;
+  flex-direction: column;
+  gap: 0.5rem;
+  margin-bottom: 1rem;
+}
+
+.input-group label {
+  font-weight: 600;
+  font-size: 0.9rem;
+}
+
+.input-group input {
+  width: 100%;
+  outline: transparent solid 2px;
+  padding: 0.2rem 1rem;
+  appearance: none;
```

Comment:
there's support issue of browsers for this, some browser offer full support while others partial so you might wanna rethink this use case

Reply:
Fixed it